.github/workflows/monorepo-validate.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
     runs-on: ${{ fromJSON(vars.RUNNER) }}
     timeout-minutes: 15
     steps:
-      - uses: browser-actions/setup-chrome@c785b87e244131f27c9f19c1a33e2ead956ab7ce # v1.7.3
+      - uses: browser-actions/setup-chrome@b94431e051d1c52dcbe9a7092a4f10f827795416 # v2.1.0
```

Comment:
It seems like from the changelog we'll end up installing the `stable` version by default instead of `latest` when you don't specify. That feels better anyways but not sure if we actually wanted `latest`